code/__DEFINES/antagonists.dm

```diff
@@ -97,6 +97,76 @@ GLOBAL_LIST_INIT(heretic_start_knowledge,list(/datum/eldritch_knowledge/spell/ba
 	WIZARD_LOADOUT_SOULTAP, \
 )
 
+/// Chance that the traitor could roll hijack if the pop limit is met.
+#define HIJACK_PROB 10
```

Comment:
This should probably be weight rather than prob chances, otherwise now you have to make sure they all add to 100, and it's such a pain.

Reply:
I guess you didn't make these, up to you while you're completely redoing objectives.